hack/env

```diff
@@ -33,4 +33,10 @@
 # NOTE:   only committed code is built.
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
+relative_bin_path="$( os::util::repository_relative_path "${OS_OUTPUT_BINPATH}" )"
```

Comment:
99% sure we do ... realistically people are using this to do builds locally pretty much 99.99999% of the time, they're not running other stuff as much.